Web/Views/Shared/_Layout.cshtml

```diff
@@ -65,7 +60,7 @@
 
     <footer class="border-top footer text-muted">
         <div class="container">
-            &copy; 2019 - Demo App
+            &copy; 2019 - Viia Demo App
```

Comment:
We probably should bump the year as well, but if I remember correctly last time I wanted to do that, Mads said we can't yet for some reason.

Reply:
@majspiir ?